id.js

```diff
@@ -15,10 +15,9 @@ Id.prototype[fl.concat] = function(b) {
 };
 
 // Monoid (value must also be a Monoid)
-Id[fl.empty] = function() {
-    return new Id(this.value[fl.empty] ? this.value[fl.empty]() : this.value.constructor[fl.empty]());
+Id.prototype[fl.empty] = function() {
+    return new Id(typeof this.value[fl.empty] === 'function' ? this.value[fl.empty]() : this.value.constructor[fl.empty]());
```

Comment:
It's sort of strange to have something in value['fantasy-land/empty'] which could not be called, I think it's better to not check and fail fast, so users will spot errors faster.